code/game/objects/items/weapons/weaponry.dm

```diff
@@ -556,3 +556,18 @@
 				bug.death(1)
 			else
 				qdel(target)
+
+/obj/item/weapon/circlegame
+	name = "circled hand"
+	desc = "If somebody looks at this while it's below your waist, you get to bop them."
+	special_examine_text = "They are holding their hand in a circle below their waist!"
+	icon_state = "madeyoulook"
+	force = 0
+	throwforce = 0
+	flags = DROPDEL
+	attack_verb = list("bopped")
+
+/obj/item/weapon/circlegame/examine_reaction(mob/living/carbon/human/owner, mob/examiner)
+	to_chat(examiner, "<span class='userdanger'>You accidentally look at [owner]'s circled hand!</span>")
+	to_chat(owner, "<span class='notice'>You catch [examiner] glimpsing at your circled hand.</span>")
+	..()

```

Comment:
I don't think this parent call works and I'm not sure why, help would be appreciated. Basically the special_examine_text doesn't show properly for the circled hand in particular, but the special_examine_text system works fine for everything else.